Doc/library/struct.rst

```diff
@@ -443,6 +443,9 @@ The :mod:`struct` module also defines the following type:
 
       The format string used to construct this Struct object.
 
+      .. versionchanged:: 3.6
+         The format string is now a Unicode string, instead of a bytes string.
```

Comment:
Maybe use just `str` and `bytes`? There is nothing specific to the Unicode standard here.